<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Download</class>
    <widget class="QWidget" name="Download">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>562</width>
                <height>733</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="NotificationBox" name="message_box">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QScrollArea" name="scrollArea">
                    <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Plain</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                        <enum>Qt::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="sizeAdjustPolicy">
                        <enum>QAbstractScrollArea::AdjustToContents</enum>
                    </property>
                    <property name="widgetResizable">
                        <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="content">
                        <property name="geometry">
                            <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>544</width>
                                <height>652</height>
                            </rect>
                        </property>
                        <layout class="QVBoxLayout" name="content_layout">
                            <item>
                                <widget class="QGroupBox" name="groupBox">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="title">
                                        <string>Filter</string>
                                    </property>
                                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                                        <item>
                                            <widget class="QComboBox" name="filter_combo"/>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="add_filter_button">
                                                <property name="text">
                                                    <string>Add Filter</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="widget" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="query_button">
                                <property name="text">
                                    <string>Query</string>
                                </property>
                                <property name="autoDefault">
                                    <bool>true</bool>
                                </property>
                                <property name="default">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>NotificationBox</class>
            <extends>QFrame</extends>
            <header>solarviewer.ui.util</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
